### PR TITLE
I submitted the Chinese version of the manual

### PR DESCRIPTION
**Pull request by kokdemo**
cover almost post in the manual

**What does this PR do and why is it necessary?**
There are a lot of users in China who need a manual to learn

**How was it tested? How can it be tested by the reviewer?**

**Any background context you want to provide?**

**What are the relevant tickets if any?**

**Screenshots (if appropriate)**

**Further notes**

### CODE REVIEW

File: i18n/zh-cn/code.json
Comment (nucleus-ffm) [inline i18n/zh-cn/code.json:1]: not yet fully translated

File: i18n/zh-cn/docusaurus-plugin-content-blog/options.json
Comment (nucleus-ffm) [inline i18n/zh-cn/docusaurus-plugin-content-blog/options.json:1]: not yet fully translated

File: i18n/zh-cn/docusaurus-plugin-content-docs/current.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "version.label": {
+    "message": "Next",
```
Comment (nucleus-ffm) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current.json:3]: Missing translation

File: i18n/zh-cn/docusaurus-plugin-content-docs/current.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  "sidebar.tutorialSidebar.category.Grundfunktionen": {
+    "message": "基础功能",
+    "description": "The label for category Grundfunktionen in sidebar tutorialSidebar"
+  },
+  "sidebar.tutorialSidebar.doc.Über": {
+    "message": "关于",
+    "description": "The label for the doc item Über in sidebar tutorialSidebar, linking to the doc about"
+  },
+  "sidebar.tutorialSidebar.doc.Mitwirkende": {
+    "message": "贡献者",
```
Comment (nucleus-ffm) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current.json:31]: The translation says "benefactor" - is that the right translation for "contributor"?

File: i18n/zh-cn/docusaurus-theme-classic/footer.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  "link.title.Mehr": {
+    "message": "更多",
+    "description": "The title of the footer links column with title=Mehr in the footer"
+  },
+  "link.title.Rechtliches": {
+    "message": "Legal",
+    "description": "The title of the footer links column with title=Rechtliches in the footer"
+  },
+  "link.item.label.Mitwirkende": {
+    "message": "贡献者",
```
Comment (Bra-Ger) [inline i18n/zh-cn/docusaurus-theme-classic/footer.json:43]: Translation sounds strange in English but might be correct in Chinese

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/Schnitt.md [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+
+### 剪切、复制和粘贴
+
+要在您的项目内剪切或复制音频,请选择要复制的项目或设置一个时间标记。然后,如果您右键单击所选区域(或打开"编辑"菜单),您可以复制整个项目(标准行为;也可通过<kbd>Cmd</kbd>/<kbd>Ctrl</kbd> + <kbd>C</kbd>)或项目内的所选区域。剪切的操作方式相同。
+
+要粘贴复制或剪切的内容,请单击左侧[音轨区域[23]](GUI-overview)并选择要将内容放置到的音轨。然后将光标置于所需位置,并通过"编辑"或上下文菜单选择"粘贴" - 或使用键盘快捷键<kbd>Cmd</kbd>/<kbd>Ctrl</kbd> + <kbd>V</kbd>。
+
+![复制和粘贴](https://raw.githubusercontent.com/Ultraschall/ultraschall-manual/main/assets/images/Schnitt/copy-and-cut.png) _上下文菜单提供了各种复制和剪切模式。_
+
+### 双端录音
```
Comment (Bra-Ger) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/Schnitt.md:201]: Retranslation into English is strange "Dual-Ended Recording" but the translation might be OK.

**Review by Bra-Ger — CHANGES_REQUESTED** [review]
I found only a few minor issues  which are caused by the retranslation into English. Please check if the translation in the Chinese version are correct.

**Review by nucleus-ffm — CHANGES_REQUESTED** [review]
looks good, but some translations sound strange

**Comment by nucleus-ffm** [comment]
Thank you for your work. We really appreciate your pull request. We have tried to review it (with a translation of course) and have noticed some strange translations. You may want to take a look at our comments and confirm that this is the correct translation. There are also some minor translations missing. 

There will also soon be the 5.1 release of Ultraschall with some minor updates to the manual [prepare 5.1](https://github.com/Ultraschall/ultraschall-manual/tree/prepare-5.1). We will merge the branch soon. It would be great if you could also update your translation then.

**Comment by nucleus-ffm** [comment]
In the meantime, we merged version 5.1 into the docu. Are you interested in updating the CH version?

**Comment by nucleus-ffm** [comment]
We will merge the PR into a new branch to test it. We will create a new PR to merge it into main afterwards.

**Pull request merged**